runtime/doc/nvim_terminal_emulator.txt

```diff
@@ -5,6 +5,7 @@
 
 
 Terminal emulator					   *terminal-emulator*
+							     *terminal-buffer*
```

Comment:
That's a nice solution. I'm not sure that I would automatically understand TUI to mean that, but if you specify that that's the way it's intended it should work.